src/cpp/shared/expansions.h

```diff
@@ -0,0 +1,18 @@
+#include <map>
+#include "combat\army.h"
+
+class stack_expansions {
```

Comment:
Let's set a convention that these names should follow the class names they're expanding. So, make this army_extra/army_expansions